wordle-web/pages/game.vue

```diff
@@ -11,7 +11,27 @@
       </v-row>
     </v-container>
 
-    <v-container v-if="isLoaded">
+  <v-container v-if="isLoaded">
+      <v-row justify="end">
+        <v-btn>
+          <v-container @click="toggleDialog">
+            {{ newName }}
+          </v-container>
+
+          <v-dialog v-model="dialog" width="450" persistent>
```

Comment:
To get the dialog to appear automatically at the end of the game, do something like this in your typescript:
```suggestion
          @Watch("this.wordleGame.state")
           function(){
                 if (this.wordleGame.state === GameState.Won) {
                   // something to check if the player name is empty or guest
                   this.dialog=true
                  }
           }
```